how-it-works/0_general/01_overview-of-the-internet-computer.subpage.md

```diff
@@ -81,11 +81,10 @@ Each subnet of the IC thereby is its own replicated state machine that makes pro
 
 ## Chain-Key and chain-evolution technology
 
-Many parts of the protocol of the IC depend on [*chain-key cryptography*](https://internetcomputer.org/how-it-works/#Chain-key-cryptography), also referred to as chain-key technology.
-Chain-key cryptography is the collection of cryptographic mechanisms that enable the decentralized operation of Internet Computer Protocol.
-[Chain-evolution technology](https://internetcomputer.org/how-it-works/#Chain-evolution-technology) refers to specific cryptography-based mechanisms that enable the IC to operate in the long term.
-For example, chain-evolution technology enables new nodes to efficiently join a subnet or nodes that have been down to efficiently catch up with the remaining part of the subnet.
-Both chain-key and chain-evolution technology sets the IC apart from other projects in terms of technology.
+Many parts of the protocol of the IC depend on [*chain-key cryptography*](https://internetcomputer.org/how-it-works/#Chain-key-cryptography), also referred to as chain-key technology. Chain-key cryptography is the collection of cryptographic mechanisms that enable the decentralized operation of Internet Computer Protocol.
+[Chain-evolution technology](https://internetcomputer.org/how-it-works/#Chain-evolution-technology) refers to specific cryptography-based mechanisms that enable the IC to operate in the long term. 
+
+For examples of concrete benefits: chain-evolution technology enables ICP canister smart contracts to call into other blockchains, or for new nodes to efficiently join a subnet or nodes that have been down to efficiently catch up with the remaining part of the subnet. Both chain-key and chain-evolution technology sets the IC apart from other projects in terms of technology.
```

Comment:
No, I don't think that is a consistent use of the terms. With "chain-evolution technology" we mean things like upgrading replica software or replacing nodes in a subnet. That's a different level. What enables canisters to call into other blockchains is _chain-key signing_ - shall we use that term here?

I think I would also drop the "efficiently join a subnet" here; it is true, but that's a very subtle point that no one will be able to understand here.